Ollivanders/src/net/pottercraft/Ollivanders2/O2Players.java

```diff
@@ -31,6 +31,7 @@
    private String masterSpellLabel = "Master_Spell";
    private String animagusLabel = "Animagus";
    private String animagusColorLabel = "Animagus_Color";
+   private String muggleLabel = "Muggle";
 
    public O2Players (Ollivanders2 plugin)
    {
```

Comment:
Somehow indenting got messed up here. The end } is not aligned with the starting and then the functions that follow are aligned with the closing }